test/libgit2.jl

```diff
@@ -130,6 +140,47 @@ mktempdir() do dir
             end
         #end
 
+        #@testset "with remote branch and refspec" begin
+            repo = LibGit2.init(cache_repo)
+            try
+                @test isdir(cache_repo)
+                @test isdir(joinpath(cache_repo, ".git"))
+
+                # set a remote branch
+                branch = "upstream2"
+                LibGit2.GitRemote(repo, branch, repo_url,"+refs/heads/$branch:refs/remotes/origin/$branch") |> finalize
+
+                config = joinpath(cache_repo, ".git", "config")
+                lines = split(open(readstring, config, "r"), "\n")
+                @test any(map(x->x == "[remote \"upstream\"]", lines))
```

Comment:
if it was of the form `any(map(f, collection))` then yeah you can replace that with `any(f, collection)`